draft-ietf-httpbis-priority.md

```diff
@@ -376,8 +376,8 @@ where to send registration requests.
 
 The Priority HTTP header field can appear in requests and responses. A client
 uses it to specify the priority of the response. A server uses it to inform
-the client that the priority was overwritten. An intermediary can use the
-Priority information from client requests and server responses to correct or
+the client that the priority that was applied. An intermediary can use
```

Comment:
```suggestion
the client and intemediaries of the priority that was applied. An intermediary can use
```
IIRC, the header in the response is primarily for the benefit of intermediaries -- a client doesn't really care what priority was actually applied, but an intermediary can consider the server's suggestion.